global_planner:
  goal_predefined: true
  path_predefined: false
  goal_x: 1.0
  goal_y: -2.5
  path_x: []
  path_y: []
  planner_params:
    floyd:  true
    bspline: true
    bspline_order: 2

map_manager:
  erode_size: 2
  inflate_size: 10
  person_inflate_size: 0.5
  group_inflate_size: 0.5

group_manager:
  predicted_steps: 2


timestep: 0.1


